ssl/statem/extensions_clnt.c

```diff
@@ -16,10 +16,37 @@ EXT_RETURN tls_construct_ctos_renegotiate(SSL_CONNECTION *s, WPACKET *pkt,
                                           unsigned int context, X509 *x,
                                           size_t chainidx)
 {
-    /* Add RI if renegotiating */
-    if (!s->renegotiate)
-        return EXT_RETURN_NOT_SENT;
+    if (!s->renegotiate) {
+        /* If not renegotiating, send an empty RI extension to indicate support */
+
+        #if DTLS_MAX_VERSION_INTERNAL != DTLS1_2_VERSION
+        # error Internal DTLS version error
+        #endif
```

Comment:
Thanks - just from a style perspective there should be no spaces before the "#", i.e. it should be this:

```suggestion
#if DTLS_MAX_VERSION_INTERNAL != DTLS1_2_VERSION
# error Internal DTLS version error
#endif
```